src/NUnitFramework/tests/Internal/RuntimeFrameworkTests.cs

```diff
@@ -317,6 +317,7 @@ public override string ToString()
             new FrameworkData(RuntimeType.NetFramework, new Version(3,5), new Version(2,0,50727), "net-3.5", "Net 3.5"),
             new FrameworkData(RuntimeType.NetFramework, new Version(4,0), new Version(4,0,30319), "net-4.0", "Net 4.0"),
             new FrameworkData(RuntimeType.NetFramework, new Version(4,5), new Version(4,0,30319), "net-4.5", "Net 4.5"),
+            new FrameworkData(RuntimeType.NetFramework, new Version(4,6), new Version(4,0,30319), "net-4.6", "Net 4.6"),
```

Comment:
The current code doesn't support reading "build" numbers from the provided `Version` so I left it like this for now but I'd appreciate some feedback on this area please.